taskcluster/ci/config.yml

```diff
@@ -21,7 +21,7 @@ taskgraph:
     repositories:
         mobile:
             name: android-components
-    cached-task-prefix: project.mobile.android-components
+    cached-task-prefix: mobile.android-components
```

Comment:
I'm seizing this opportunity to put the cached tasks under `mobile.`. As matter of fact, this is the only index we currently have under https://firefox-ci-tc.services.mozilla.com/tasks/index/project.mobile.android-components

Reply:
Sounds good. So from now on we want to consider "mobile" as standalone rather than under the `project`/releng umbrella or is this (the `project` prefix removal) something that has happened already for most of the stuff and I haven't noticed?

Reply:
Yeah, @tomprince and @escapewindow told `project` should be deprecated now that we have the `firefox-ci` TC instance. Not a lot has been done yet. Work started with https://phabricator.services.mozilla.com/D67177.